indico/modules/rb_new/client/js/common/bookings/BookingDetailsModal.jsx

```diff
@@ -0,0 +1,263 @@
+        let interval = null;
+        if (repeatFrequency === 1) {
+            type = 'daily';
+        } else if (repeatFrequency === 2) {
+            type = 'every';
+            interval = 'week';
+            number = repeatInterval;
+        } else if (repeatFrequency === 3) {
+            type = 'every';
+            interval = 'month';
+            number = repeatInterval;
+        }
+        return ({type, number, interval});
+    };
+
+    _getRowSerializer(day) {
+        const {booking: {room}} = this.props;
+        return (occurrences) => ({
+            availability: {
+                bookings: occurrences[day].map((candidate) => ({...candidate, bookable: false})) || [],
+            },
+            label: moment(day).format('L'),
+            key: day,
+            room
+        });
+    }
+
+    renderTimeline = (occurrences, dateRange) => {
+        const hourSeries = _.range(6, 22, 2);
+        const rows = dateRange.map((day) => this._getRowSerializer(day)(occurrences));
+        return <DailyTimelineContent rows={rows} hourSeries={hourSeries} />;
+    };
+
+    renderBookingHistory = (editLogs, createdOn, createdBy) => {
+        const createdDate = serializeDate(toMoment(createdOn), 'L');
+        const items = (editLogs.map((log, i) => {
+            const {timestamp, info, userName} = log;
+            const basicInfo = <strong>{info[0]}</strong>;
+            const details = (info[1] ? info[1] : null);
+            const logDate = serializeDate(toMoment(timestamp), 'L');
+            if (details) {
+                const popupContent = <span styleName="popup-center"> {details}</span>;
+                const wrapper = <PopupParam content={popupContent} />;
+                return (
+                    <List.Item key={i}>
+                        <Translate>
+                            <Param name="date" value={logDate} wrapper={<span styleName="log-date" />} />
+                            {' - '}
+                            <Param name="info" wrapper={wrapper} value={basicInfo} /> by
+                            {' '}
+                            <Param name="user" value={userName} />
+                        </Translate>
+                    </List.Item>
+                );
+            }
+            return (
+                <List.Item key={i}>
+                    <Translate>
+                        <Param name="date" value={logDate} wrapper={<span styleName="log-date" />} />
+                        {' - '}
+                        <Param name="details" value={basicInfo} /> by
+                        {' '}
+                        <Param name="user" value={userName} />
+                    </Translate>
+                </List.Item>
+            );
+        }));
+        return (
+            <div styleName="booking-logs">
+                <Header><Translate>Booking history</Translate></Header>
+                <List divided styleName="log-list">
+                    {items}
+                    <List.Item>
```

Comment:
what about injecting the fake log item on the server side? you could append it `post_dump` in the `ReservationDetailsSchema` (assuming you move log handling there). if you prefer to do it on the client:

    editLogs = [...editLogs, {your fake log entry here}];

Reply:
I am no sure about it, I would need to fake some id there

Reply:
what about `"created"`? Doesn't need to be a number ;)

Reply:
yeah, okay, that makes sense :)